data-prepper-plugins/common/src/main/java/org/opensearch/dataprepper/plugins/sink/FileSink.java

```diff
@@ -54,12 +55,14 @@ public FileSink() {
     public FileSink(final String outputFile) {
         this.outputFilePath = outputFile == null ? SAMPLE_FILE_PATH : outputFile;
         isStopRequested = false;
+        initialized = false;
```

Comment:
It might be better to move the code in between these two changes into the `initialize()` method. This will make it a better example for any contributors who look at how the code works.

Reply:
Made the changes as suggested.